grid_map_rviz_plugin/include/grid_map_rviz_plugin/modified/frame_manager.h

```diff
@@ -192,9 +192,11 @@ class FrameManager : public QObject
   template <class M>
   void registerFilterForTransformStatusCheck(tf2_ros::MessageFilter<M>* filter, Display* display)
   {
-    filter->registerCallback(boost::bind(&FrameManager::messageCallback<M>, this, _1, display));
+    filter->registerCallback(boost::bind(&FrameManager::messageCallback<M>, this,
```

Comment:
With c++14 it is recommended to use lambdas, not?

Reply:
This seems like a smaller change for right now, would prefer to get this in and look into other options later, and I think still is backwards compatible back to c++11 if that is a concern for any other users (but possibly lambdas are too?  Would have to look into it).